src/main/scala/io/citrine/lolo/Learner.scala

```diff
@@ -1,27 +1,32 @@
 package io.citrine.lolo
 
-/**
-  * Created by maxhutch on 11/14/16.
-  */
+import io.citrine.random.Random
```

Comment:
This is not an issue with this package. But should the classpath from SpRandom be `io.citrine.sprandom` instead of `io.citrine.random` ?

Reply:
.....yeah, it probably should